FLT/DedekindDomain/AdicValuation.lean

```diff
@@ -275,10 +275,26 @@ open IsLocalRing in
 /-- The canonical isomorphism from A / v to 𝓞ᵥ / v, where 𝓞ᵥ is the integers of the
 completion Kᵥ of the field of fractions K of A. -/
 noncomputable def ResidueFieldEquivCompletionResidueField :
-    A ⧸ v.asIdeal ≃+* ResidueField (v.adicCompletionIntegers K) :=
-  RingEquiv.ofBijective (ResidueFieldToCompletionResidueField K v)
-  -- issue FLT#449
-    ⟨Ideal.quotientMap_injective' <| ge_of_eq Ideal.LiesOver.over, sorry⟩
+    A ⧸ v.asIdeal ≃+* ResidueField (v.adicCompletionIntegers K) := by
+  apply RingEquiv.ofBijective (ResidueFieldToCompletionResidueField K v)
+    ⟨Ideal.quotientMap_injective' <| ge_of_eq Ideal.LiesOver.over, ?_⟩
+
+  intro z
+  obtain ⟨x, hx⟩ :=
+    Submodule.Quotient.mk_surjective (p := maximalIdeal ↥(adicCompletionIntegers K v)) z
+  rw [← hx, Ideal.Quotient.mk_eq_mk]
+  suffices ∃ a : A, (ResidueFieldToCompletionResidueField K v) a = Ideal.Quotient.mk _ x by
+    obtain ⟨a, ha⟩ := this
+    refine ⟨a, ?_⟩
+    rw [← ha]
+    rfl
+  change ∃ a, Ideal.Quotient.mk (maximalIdeal ((v.adicCompletionIntegers K))) _ = _
+  simp_rw [Ideal.Quotient.mk_eq_mk_iff_sub_mem, mem_maximalIdeal, mem_nonunits_iff]
+  -- TODO - figure out why this can't be 'simp_rw/simp'
```

Comment:
Good question! If you can minimise you could ask on the Zulip. My guess is that unification isn't smart enough to figure out what `v` is?